source/agora/consensus/data/DataPayload.d

```diff
@@ -177,24 +177,25 @@ unittest
 }
 
 // HashFull test
-unittest
-{
-    // https://tools.ietf.org/html/rfc7693#appendix-A
-    static immutable ubyte[] hdata = [
-        0xBA, 0x80, 0xA5, 0x3F, 0x98, 0x1C, 0x4D, 0x0D, 0x6A, 0x27, 0x97, 0xB6,
-        0x9F, 0x12, 0xF6, 0xE9,
-        0x4C, 0x21, 0x2F, 0x14, 0x68, 0x5A, 0xC4, 0xB7, 0x4B, 0x12, 0xBB, 0x6F,
-        0xDB, 0xFF, 0xA2, 0xD1,
-        0x7D, 0x87, 0xC5, 0x39, 0x2A, 0xAB, 0x79, 0x2D, 0xC2, 0x52, 0xD5, 0xDE,
-        0x45, 0x33, 0xCC, 0x95,
-        0x18, 0xD3, 0x8A, 0xA8, 0xDB, 0xF1, 0x92, 0x5A, 0xB9, 0x23, 0x86, 0xED,
-        0xD4, 0x00, 0x99, 0x23
-    ];
-    const abc_exp = Hash(hdata);
-
-    DataPayload data_payload = DataPayload(cast(ubyte[])"abc");
-    assert(hashFull(data_payload) == abc_exp);
-}
+// TODO: do we really need this test here which was copied from Hash.d - (it will currently fail if enabled)
```

Comment:
What shall we do with this test?

Reply:
Remove

Reply:
Shall I remove or change the expectation in this test?

Reply:
Remove the whole test, it's not `DataPayload`'s role to check the result of hashing like this.

Reply:
As agreed verbally I will remove this test from here as it doesn't need to test Hashing here.